### DISCUSSION TITLE
Dowload fails for a website that used to work for me

### DISCUSSION THREAD

**Discussion by chaimziman** [aclap-dev/video-downloadhelper#1349] [Help and Q&A]
### Your question

I have been dowloading video off of a website for a long time called torahanytime.com and now it is not working. here is a link. https://torahanytime.com/lectures/326830

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"d22bf807bb194d6f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\zimanc\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 7,
    "successfull_dl": 65,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 24H2 build 26100.2033, Firefox 131.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

When I tried to get VDH to download your video, I got about a minute of the 23+ minute video.  I looked in the Network Monitor & did find an HLS master manifest.  But when I ran ffprobe against it, this is what I got:

[ffprobe.txt](https://github.com/user-attachments/files/17348444/ffprobe.txt)

It looks to me like there's a combination of things going on here.  On one hand the manifest & the things it relies on do not seem to be contructed according to standard.  On the other hand at least some of this content is protected against reading, which is what the 403 Forbidden error indicates.

This seems to be the case for this video.  Are other videos on the site also giving you trouble?  It could be that the web site changed the way they're doing things, or at least tried to, & they're still learning how to do it right.  They're definitely not doing it right now.

**Comment by xezoxeon** [comment]
I am having same problem was able to downloads videos a couple days ago now I cant download them :(( I want a refund...

**Reply by Mattmjs** [reply]
From the same site above ?

**Reply by xezoxeon** [reply]
No

**Reply by Mattmjs** [reply]
A license doesn't automatically guarantee that it will work. But if you want a refund open a separate discussion on that.

**Reply by xezoxeon** [reply]
No worries I don't actually need one.. I paid to support the company got some use... Just thought it would last longer. surly they will fix it

**Reply by Mattmjs** [reply]
It would need to be investigated, another separate discussion on that. Sites sometimes use DRM, encryption or other non standard
methods to stop downloading.

**Reply by xezoxeon** [reply]
Other's are experiencing too its not just the website.. something changed

**Comment by Mattmjs** [comment]
There is a download button placed on the page, it says sign in to download lecture. Maybe that's an option if they allow it.

I think when the download starts it's already slow, terribly throttled to the point that the download terminates.
But if you really want this there is a way. I saved a html document from the site from the developers tools Network , it's the first entry that appears. There is whole bunch of code in there but all you need to do is use the find function for mp4 or m3u8 in notepad.

If you copy the link the way it is and paste it in a new tab like I did it'll say _sorry we're having a little trouble_. There are two instances
of `\u0026` in the link. You can replace those with &. This makes the link work. But there is a better option.

![lecture site](https://github.com/user-attachments/assets/4240f4cd-6292-4e7e-8808-fd1720c02566)

Keep using the find function for mp4 and you find another option that doesn't need to be modified. Copy and paste this one in a new tab
and it plays. Now you can download either with VDH or the browser itself using right click save video as.

![lecture_1](https://github.com/user-attachments/assets/52c3b47a-0f6e-40a4-b389-56940aa43128)

**Comment by xezoxeon** [comment]
"Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)."

**Comment by 3ifish** [comment]
Just to add my name to the increasing list of Video DownloadHelper users who have not been able to download media.  Same as many others, sites and files that did download not longer are able to be downloaded.

I'm sure the issues will be corrected (I'm sure apps like this are playing a sort of Wack A Mole game with the various sites).  Just wanted to add another voice to the crowd.

**Reply by xezoxeon** [reply]
Actually a similar site "soap2day" allowed a right click and download on any video :) good old days of 2022

**Reply by 3ifish** [reply]
Doesn't seem to be working for me (wish I could say otherwise).  The Whac-A-Mole game continues.

**Reply by xezoxeon** [reply]
https://soaper.tv/movie_dVGLaymkYb.html still fails as of today

**Reply by Mattmjs** [reply]
@3ifish open a new discussion and post your information along with one or more pages where the problem is happening.

@xezoxeon , it still needs a fix from the developer, if you don't want to wait I tested it in VLC media player and it works.
 Use copy url in VDH then use it VLC. Look online how to use VLC for downloading.

**Reply by 3ifish** [reply]
@Mattmjs 
I can not download videos from ANY site at this point

Tried your suggestion using VLC, but did not work for me.  Tried a couple of videos, same results - a continuous stream of error messages telling me File Exists, I will overwrite if I continue...

Thanks

**Reply by Mattmjs** [reply]
@3ifish  My VLC suggestion was for the other user, it does work but it's not a perfect solution for all sites.
As I said you should open a new discussion if it isn't working for you, share examples and other users like myself can test
if the same happens or it's just you. Just saying it's not working on any site is not helpful, this does not resolve anything.
You should also look to see if anti virus and firewall are not blocking the companion app.
